manywheel/build.sh

```diff
@@ -15,7 +15,7 @@ case "${GPU_ARCH_TYPE:-BLANK}" in
     rocm)
         bash "${SCRIPTPATH}/build_rocm.sh"
         ;;
-    cpu | cpu-cxx11-abi)
+    cpu | cpu-cxx11-abi | cpu-s390x)
```

Comment:
Should `cpu-s390x` be removed from here too?

Reply:
No, with this change, I can remove commit https://github.com/pytorch/pytorch/pull/120398/commits/2928b2c4d1bfa0ac03881b87cb965432ac381da2

Or I can also drop this change and keep it as just "cpu" for s390x. But for aarch64 "cpu-aarch64" is used, so I did similar "cpu-s390x" for s390x.